conf/messages

```diff
@@ -157,6 +160,9 @@ group.manage.confirm.updated.team-members.p=The team members you selected can vi
 group.manage.delete=Delete group
 group.manage.h1=Manage access groups
 group.manage.members=Manage team members
+group.manage.members.h1=Manage team members’ access groups
+group.manage.members.p=You can view team members’ details here and see what access groups they are in. You can also <a href="{0}">start adding team members to your agent services account</a>.
```

Comment:
I would avoid having html in the messages, have it inline the view if you need.

Also, when you're using this in the view `@p("group.manage.members.p")` you don't have an argument 👀